src/TurbulenceClosures/turbulence_closure_diagnostics.jl

```diff
@@ -2,18 +2,16 @@
 ##### Timescale for diffusion across one cell
 #####
 
-using Oceananigans.Grids: topology
+using Oceananigans.Grids: topology, min_Δx, min_Δy, min_Δz
 
 function min_Δxyz(grid)
-    topo = topology(grid)
-
-    Δx = minimum_grid_spacing(grid.Δx, topo[1])
-    Δy = minimum_grid_spacing(grid.Δy, topo[2])
-    Δz = minimum_grid_spacing(grid.Δz, topo[3])
-
-    return min(Δx, Δy, Δy)
+    Δx = min_Δx(grid)
```

Comment:
is `minimum_grid_spacing` used for anything else after this change?  If not that might be something we can remove.

Reply:
I already removed it from `turbulence_closure_diagnostics`, no? Unless I missed something. 

But yes, we don't need that anymore. I'm also removing it from `cell_advection_timescale`.

Reply:
Agreed